scripts/modules/events/integration-manager-di.js

```
@@ -0,0 +1,1375 @@
+						error.message.includes('timeout'))
+				) {
+					throw error; // Don't retry these errors
+				}
+
+				if (attempt === maxRetries) {
+					throw error; // Final attempt failed
+				}
+
+				// Wait before retry with short delays for testing
+				const baseDelay = this.config.retryDelay || 100; // Short delay for tests
+				const delay = Math.min(baseDelay * Math.pow(2, attempt), 1000);
```

Biome (1.9.4)

[error] Use the '**' operator instead of 'Math.pow'.

Unsafe fix: Use the '**' operator instead of 'Math.pow'.


(lint/style/useExponentiationOperator)